sources/packages/backend/libs/integrations/src/services/disbursement-schedule/disbursement-schedule.service.ts

```diff
@@ -199,4 +200,68 @@ export class DisbursementScheduleService extends RecordDataModelService<Disburse
     const repository = disbursementScheduleRepo ?? this.repo;
     return repository.update({ id: In(disbursementIds) }, { dateSent });
   }
+
+  /**
+   * Fetch the eligible COEs for a given date.
+   * @param generatedDate Date in which the eligible COE is required for an institution.
+   * @returns eligible COEs.
+   */
+  async getEligibleCOE(
+    generatedDate?: string,
```

Comment:
The method can receive the type that it needs, for instance, a `Date`. The consumer method can do the conversion.